components/prism-sas.js

```diff
@@ -1,3 +1,29 @@
+(function (Prism) {
+
+	numberPattern = /\b(?:\d[\da-f]*x|\d+(?:\.\d+)?(?:e[+-]?\d+)?)\b/i;
```

Comment:
```suggestion
	var numberPattern = /\b(?:\d[\da-f]*x|\d+(?:\.\d+)?(?:e[+-]?\d+)?)\b/i;
```

Same for the other variables.